examples/js/loaders/GLTFLoader.js

```diff
@@ -1547,59 +1523,57 @@ THREE.GLTFLoader = ( function () {
 
 	}
 
-	function isArrayEqual( a, b ) {
+	function createGeometryKey( geometry ) {
 
-		if ( a.length !== b.length ) return false;
+		var dracoExtension = geometry.extensions && geometry.extensions[ EXTENSIONS.KHR_DRACO_MESH_COMPRESSION ];
+		var geometryKey;
 
-		for ( var i = 0, il = a.length; i < il; i ++ ) {
+		if ( dracoExtension ) {
 
-			if ( a[ i ] !== b[ i ] ) return false;
+			geometryKey = 'draco:' + dracoExtension.bufferView
+				+ ':' + dracoExtension.indices
+				+ ':' + createAttributesKey( dracoExtension.attributes );
+
+		} else {
+
+			geometryKey = geometry.indices + ':' + createAttributesKey( geometry.attributes );
```

Comment:
I realize it wasn't covered in the original method, but for good measure could you include `primitiveDef.mode` here? It applies only to the non-Draco case.

Reply:
As far as I understand, that is a Enum right? So I can simply concatenate it?

Reply:
Yep!